src/docfx/build/metadata/UserMetadata.cs

```diff
@@ -19,6 +20,9 @@ internal class UserMetadata
         [JsonProperty("monikerRange")]
         public SourceInfo<string> MonikerRange { get; private set; } = new SourceInfo<string>("");
 
+        [JsonConverter(typeof(OneOrManyConverter))]
+        public SourceInfo<string>[] Monikers { get; private set; } = Array.Empty<SourceInfo<string>>();
```

Comment:
```suggestion
        public SourceInfo<SourceInfo<string>[]> Monikers { get; private set; }
```

Reply:
If change the type as suggested, `[JsonConverter(typeof(OneOrManyConverter))]` would not work

Reply:
OK, then I'm good with the current approach